proof/refine/AARCH64/Schedule_R.thy

```diff
@@ -249,9 +249,23 @@ lemma switchLocalFpuOwner_corres[corres]:
    apply (auto simp: current_fpu_owner_Some_tcb_at fpuOwner_asrt_cross)
   done
 
+(*FIXME FPU: move tcbFlag lemmas*)
```

Comment:
FIXME

Reply:
If we leave it, then yes to removing FPU flag from FIXME.
Generally, I'm not opposed to having the flag lemmas in Tcb_R for the moment. Here we are in Schedule_R, and you felt like these don't belong here. Maybe if you can expand your thought on why they don't belong, a place will pop up?

In CRefine, we often put these kinds of things into SR_Lemmas, I guess Invariants_H is one possible place, StateRelation another. Or if it's nearly all generic (or with identical derivations in Arch), then having a `TcbFlags_R.thy` theory might be warranted. I'm not sure myself, having not worked with these lemmas closely.